Access Engineering.js

```diff
@@ -97,7 +97,8 @@ function scrape(doc, url) {
 		if (edition) item.edition = edition;
 
 		// Author
-		let author = ZU.xpath(doc, '//ul[@class="contributor-list"]//li//a');
+		// Some of old pages not having firstname, lastname seperation in markup and ignore if not
+		let author = ZU.xpath(doc, '//ul[@class="contributor-list"]//li[@data-firstnames]');
 		if (author.length > 0) {
```

Comment:
Yes, this is required. Because of some of old page which do not have content for author in markup service. https://www.accessengineeringlibrary.com/content/video/V4005352521001